src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerEndPoint.class.php

```diff
@@ -545,7 +545,7 @@ public function GetHiddenFieldsHook()
     {
         $oGlobal = TGlobal::instance();
         $aAdditionalParameterData = array();
-        $aAdditionalParameters = array('sRestrictionField', 'sRestriction', 'bIsLoadedFromIFrame');
+        $aAdditionalParameters = array('sRestrictionField', 'sRestriction', 'bIsLoadedFromIFrame', 'field');
```

Comment:
Is field different from sRestrictionField?

Reply:
Yes - but it's not obvious from the naming.

* `sRestrictionField` contains the name of the field that contains the restriction on the target table (The field pointing back to the 'parent')
* `field` contains the name of the field on the 'parent record' of which the children are being edited

Basically, if you have a `article` that has an MLT relation to `category` in the field `special_categories`

* `sRestrictionField` will be `article_id` on `category` (even if that field doesn't "really" exist)
* `field` will refer to `special_categories` on `article`